assignment.js

```diff
@@ -66,6 +76,19 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const numbers = [3, 48, 29, 12, 18]
+let currentLargestNumber = numbers[0]
+
+for (let i = 0; i <= numbers.length; i++){
```

Comment:
@justinbudlow  nice use of a `for` loop here. Also nice use of using your knowledge of arrays to use `i` as an index to implement the logic of keeping track of the largest number.